### DISCUSSION TITLE
Read-only thermostat - still useful?

### DISCUSSION THREAD

**Discussion by dcleaver** [emsesp/EMS-ESP32#1404] [Q&A] [answered]
According to the support page, I have a read-only thermostat (a Worcester-Bosch Comfort II RF). At first I was thinking this was a no-go and I'd have to replace it with a read/write thermostat. But it got me thinking - can I just use the control as a dumb-thermostat and let the EMS-ESP32 control the boiler? Will the EMS-ESP32 be able to do that? (I have a Worcester-Bosch 32CDi which I believe is fully supported)

So I'd set the thermostat to "always off" and get Home Assistant to read the current temperature. When the temperature drops, use the EMS-ESP32 to "call for heat" on the boiler. And if HA is down, I can still override manually.

Or do I have to get a read/write thermostat and set the desired temperature there?

**Comment by proddy** [comment, marked as answer]
Have a read of https://emsesp.github.io/docs/FAQ/#can-ems-esp-simulate-a-thermostat

**Reply by dcleaver** [reply]
Thanks. One of the Git issues mentioned in there goes into quite a bit of detail [https://github.com/emsesp/EMS-ESP32/issues/151](https://github.com/emsesp/EMS-ESP32/issues/151)

The main thing that's been answered is that I can manually set the boiler to fire, although it might start fighting with the thermostat setting it to off. But there's now code in there that will continuously "fight back" and keep the boiler going for as long as directed.

I'll mark this as the answer and have a play once I receive my unit.
